app/controllers/api/items_controller.rb

```diff
@@ -0,0 +1,74 @@
+module API
+  class ItemsController < ApplicationController
+    before_filter :validate_item, only: [:create, :update]
+    before_filter :authorize_token
+    before_filter :my_load_and_authorize_resource, only: [:update, :destroy, :show]
+
+    attr_reader :item
+
+    def create
+      head :created, location: sufia.api_item_path(actor.create_file_from_item)
+    end
+
+    def update
```

Comment:
@mjgiarlo since Arkivo needs to keep track of the Zotero key to Sufia id mapping, can we return 404 on `update` and `delete` if the given id does not exist? The current response is 500 if I am not mistaken, but it will be important for Arkivo to know, for example, that an `update` failed because the item did not exist (as opposed to other errors) in order for it to send a `create` request instead. Similarly, that a `delete` failed because the item was already removed (hence the error can be ignored).